FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/Demos.kt

```diff
@@ -138,7 +138,7 @@ val V1DEMO = arrayListOf(
     Demo(AVATAR_GROUP_VIEW, AvatarGroupViewActivity::class),
     Demo(BASIC_INPUTS, BasicInputsActivity::class),
     Demo(BOTTOM_NAVIGATION, BottomNavigationActivity::class),
-    Demo(BOTTOM_SHEET, BottomSheetActivity::class),
+    Demo(BOTTOM_SHEET, BottomSheetActivity::class, Badge.APIBreak),
```

Comment:
I dont think this is an API break. There wont be any new changes when this fix is introduced to users right?

Reply:
Yes, Turning it into _Modified_.